tools/Makefile

```diff
@@ -1,12 +1,18 @@
+UNAME := $(shell uname)
```

Comment:
instead of getting the uname again it can be reused from the previous usage

Reply:
Confirmed that the `UNAME` define is passed from `Makefile` in root. Removed it from `tools/Makefile` in ee30215b8fb383d27a3d46936f55936b92d70fdd.